FormFactory/CommentFormFactoryInterface.php

```diff
@@ -23,10 +23,11 @@
     /**
      * Creates a comment form
      *
-     * @param mixed $method
-     * @param array $options
+     * @param mixed $data
+     * @param array $options => method
+     * @param string $name_suffix
      *
      * @return FormInterface
      */
-    public function createForm($data = null, $options = array());
+    public function createForm($data = null, $options = array(), $name_suffix = null);
```

Comment:
I guess that the BC break (Backward Compatibility break, right ?) is on all interfaces ? but only on interfaces, not on the implementation classes ?
So, can I just remove the $name_suffix in the interface function prototype  ?

Sorry I'm new about making pull request and sharing code like this, but I'm glad to learn

thank for your quick feedback.